app/models/question.rb

```diff
@@ -2,4 +2,6 @@ class Question < ApplicationRecord
   has_many :game_questions
   has_many :games, through: :game_questions
   validates :clue, :answer, :category, :clue_id, :value, :category_id, presence: true
+
+  enum status: [:valid_quesiton, :invalid_quesiton]
```

Comment:
What makes a question valid or invalid?

Is this associated with the "value" column?